servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java

```diff
@@ -29,11 +33,20 @@
 public interface HttpLoadBalancerFactory<ResolvedAddress>
         extends LoadBalancerFactory<ResolvedAddress, FilterableStreamingHttpLoadBalancedConnection> {
 
+    @Deprecated
```

Comment:
I realized it's impossible to get rid of the deprecated implementation. We should add default implementations to base interface and this one too to be able to remove the deprecated variant.

Reply:
We discussed this when you originally introduced the change in https://github.com/apple/servicetalk/pull/1831#discussion_r712703689
Because the parent interface implements `@FunctionalInterface`, we can not make both methods `default`. We will make swap `default` impl in 0.42 (I will submit a follow-up PR, because this PR is going to 0.41 as well), and remove the old method in 0.43.

Reply:
Ah, right. Thanks for reminding me.